.github/scripts/config.sh

```diff
@@ -53,6 +53,8 @@ TESTS_PACKAGE_NAME=${TESTS_PACKAGE_NAME:-$TOOLCHAIN_NAME-tests.tar.gz}
 FFMPEG_PATH=${FFMPEG_PATH:-~/ffmpeg}
 FFMPEG_TESTS_PATH=${FFMPEG_TESTS_PATH:-~/ffmpeg-tests}
 
+CYGWIN_PATH=${CYGWIN_PATH:-~/cygwin}
```

Comment:
maybe we need to specify target in cygwin name.
I guess cygwin for x64 should be built as validation.

Reply:
Good point.